interactive/admin.py

```diff
@@ -58,7 +59,9 @@ def response_change(self, request, obj):
             self.message_user(
                 request, f"The request for {obj.full_name} has been approved"
             )
-            return HttpResponseRedirect(".")
+            return HttpResponseRedirect(
+                reverse("admin:interactive_registrationrequest_changelist")
+            )
```

Comment:
You can use the `redirect` shortcut directly here instead of the redirect and reverse (it does that under the hood for you!)